### PR TITLE
CVE-2014-3166

### PR DESCRIPTION
**Pull request by Davidjwkinghorn**
Submitting in two separate pull requests because YAML files are edited in GitHub. (I haven't yet figured out if I can edit multiple files and put them it one pull request)

### CODE REVIEW

File: cves/CVE-2014-3166.yml [hunk 1]
```diff
@@ -57,18 +68,19 @@ fixes_vcc_instructions: |
...
 - :commit: 8d60aa54abe0517d756c9d625ece75feabed613a
-  :note: ''
-- :commit: debe0fde0c897da880278fba6d4fbde05c927094
-  :note: ''
+  :note:
 vccs:
-- :commit:
-  :note: 
+- :commit: 9e1bdd3f9785ed8a84a137c98b781ddaec41b080
+  :note:
```
Comment (daviematz) [inline cves/CVE-2014-3166.yml]: You should fill in the note here with a quick description of what is happening at that commit

File: cves/CVE-2014-3166.yml [hunk 2]
```diff
@@ -22,17 +22,28 @@ announced: 2014-08-13 00:57:12.613000000 -04:00
...
+  domains to access sensitive information. Pooling with multiple domains allows
+  an attacker to bypass Public Key Pinning (PKP) on sites. By bypassing PKP on
+  sites this opens up an information disclosure vulnerability. This is due to
+  non-adherence to the principals of SPDY connections. Examples given show that
+  it was possible for a malicious actor to pretend to be any website by
+  exploiting this vulnerability. For example, the reporter provides a
+  screenshot, demonstrating that www.twitter.com was displaying their own
+  content rather than the real twitter site.
+
+  Google Chrome versions before 36.0.1985.143 were affected on Windows, OS X,
```
Comment (daviematz) [inline cves/CVE-2014-3166.yml]: You should 'Remove references to versions' like these here

File: cves/CVE-2014-3166.yml [hunk 3]
```diff
@@ -22,17 +22,28 @@ announced: 2014-08-13 00:57:12.613000000 -04:00
...
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  Provided with this vulnerability, attackers would be able to leverage multiple
+  domains to access sensitive information. Pooling with multiple domains allows
+  an attacker to bypass Public Key Pinning (PKP) on sites. By bypassing PKP on
+  sites this opens up an information disclosure vulnerability. This is due to
+  non-adherence to the principals of SPDY connections. Examples given show that
```
Comment (daviematz) [inline cves/CVE-2014-3166.yml]: You should remove or explain the 'principals of SPDY connections', it isn't something that outsiders to Chromium would understand.

File: cves/CVE-2014-3166.yml [hunk 4]
```diff
@@ -57,18 +68,19 @@ fixes_vcc_instructions: |
...
-  :note: 
+- :commit: 9e1bdd3f9785ed8a84a137c98b781ddaec41b080
+  :note:
+- :commit: eed749f90554fcb7f561c935bcee60b8e1235de2
+  :note:
+- :commit: 11c0587044d54aeef5a7e1e0e2c5fedbe55106cb
+  :note:
+- :commit: e6d017658f1f60b06eebff1b0ac5ba6751b8daa7
+  :note:
 upvotes_instructions: |
```
Comment (daviematz) [inline cves/CVE-2014-3166.yml]: +3 upvotes

File: cves/CVE-2014-3166.yml [hunk 5]
```diff
@@ -57,18 +68,19 @@ fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
-- :commit: 5db452206ce2503815abe55878179b2399cc906a
-  :note: ''
 - :commit: 8d60aa54abe0517d756c9d625ece75feabed613a
-  :note: ''
-- :commit: debe0fde0c897da880278fba6d4fbde05c927094
-  :note: ''
+  :note:
```
Comment (wylaris) [inline cves/CVE-2014-3166.yml]: Are there not any notes for the commit?  Is there a commit message that could be included?

File: cves/CVE-2014-3166.yml [hunk 6]
```diff
@@ -79,87 +91,92 @@ unit_tested:
...
+  answer: A good portion of the work was comitted here.
   commits:
-  - commit:
-    note:
-  - commit:
-    note:
+  - commit: 8d60aa54abe0517d756c9d625ece75feabed613a
+    note: |
+      Here some logic was centralized and some checks were added. PKPs were
+      changed to check to use TransportSecurityState::CheckPublicKeyPins. Checks
```
Comment (wylaris) [inline cves/CVE-2014-3166.yml]: Try to explain what TransportSecuirtyState is and how it affected the program.

File: cves/CVE-2014-3166.yml (hunk 4 above)
Comment (Nitron96) [reply cves/CVE-2014-3166.yml]: +2 upvotes

File: cves/CVE-2014-3166.yml (hunk 2 above)
Comment (Davidjwkinghorn) [reply cves/CVE-2014-3166.yml]: Will do.

File: cves/CVE-2014-3166.yml (hunk 1 above)
Comment (Davidjwkinghorn) [reply cves/CVE-2014-3166.yml]: This commit has been removed since it is not a VCC.

**Pull request merged**